### DISCUSSION TITLE
telemetry

### DISCUSSION THREAD

**Discussion by marco-santulli** [Budibase/budibase#3806] [Help]
Hi,

does budibase make any calls for telemetry / other reasons out to internet servers?

M

**Comment by mjashanks** [comment]
Hi @marco-santulli 

In self-host we do record telemetry. You can disable this with an environment variable: `ENABLE_ANALYTICS`.

This can be found in your `hosting.properties` or `.env` file, depending on your setup.

Budibase also downloads a javascript library and fetches templates from the public internet.

**Reply by marco-santulli** [reply]
hi Michael,

thanks for the answer.  I have made the changes, but I think it would be good to add to the documentation somwhere:
1. this flag, with instructions (is it with true/false? is it "true"/"false"?
2. since budibase auto-updates, I think some of these external servers need to be made accessible.  In an enterprise world machines are often hardened and firewalls blocked by default with only some white-listings.  So what are the sites that need whitelisting to endure budibase works properly?

thank you

M

**Reply by mjashanks** [reply]
I agree - we used to have a setting for this in the UI, but lost it along the way - we should add it back in. Have logged: https://github.com/Budibase/budibase/issues/3815

**Whitelisting**
prod-budi-templates.s3-eu-west-1.amazonaws.com
https://fonts.googleapis.com

On setup/update:
budibase.docker.scarf.sh
dockerhub.io

**Reply by marco-santulli** [reply]
Hi Mike,

excellent that sounds like a good idea.  Are you suggesting to add the domains to be whitelisted in the organization page too or in the documentation?

M

**Reply by mjashanks** [reply]
The whitelist was for your own reference - but yes also for documentation.

**Reply by marco-santulli** [reply]
thank you very much!!
